document/core/appendix/gen-index-instructions.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
```

Comment:
Maybe add a comment about why this was added rather than directly maintaining the rst file?

Reply:
Done.